superset/translations/pt_BR/LC_MESSAGES/messages.po

```diff
@@ -1249,11 +1249,11 @@ msgstr "Selecione uma fonte de dados"
 #: superset/assets/javascripts/explore/components/controls/DatasourceControl.jsx:132
 #: superset/assets/javascripts/explore/components/controls/VizTypeControl.jsx:120
 msgid "Search / Filter"
-msgstr "Pesquisa / Filtro"
+msgstr "Buscar / Filtrar"
```

Comment:
Yes, Search is Pesquisar and Filter is Filtrar. It's right, because we maintain both in the infinitive, it's commonly used in buttons and labels here.